Src/BootCamp.Chapter/Account.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public class Account
+    {
+        private readonly string _name;
+        private readonly decimal[] _balance;
+        private readonly string[] _accountArray;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_accountArray' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5054323)